index.html

```diff
@@ -0,0 +1,1561 @@
+                />
+              </svg>
+            </a>
+          </li>
+          <li>
+            <a href="#"
+              ><svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="16"
+                height="16"
+                viewBox="-7.5 992.5 16 16"
+                id="rss--dark"
+                x="800"
+                y="800"
+              >
+                <circle cx="-5.5" cy="1006.5" r="2" />
+                <path
+                  d="M-7.5 995.5c7.2 0 13 5.8 13 13h3c0-8.8-7.2-16-16-16v3zm0 3v3c3.9 0 7 3.101 7 7h3c0-5.5-4.5-10-10-10z"
+                />
+              </svg>
+            </a>
+          </li>
+          <li>
+            <a href="#"
+              ><svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="16"
+                height="16"
+                viewBox="-7.5 992.5 16 16"
+                id="mail--dark"
+                x="560"
+                y="560"
+              >
+                <path
+                  d="M.5 1001.5l-8-6c0-.555.446-1 1-1h14c.554 0 1 .445 1 1l-8 6zm0 1.798l8-6.001v8.203c0 .555-.446 1-1 1h-14c-.554 0-1-.445-1-1v-8.203l8 6.001z"
+                />
+              </svg>
+            </a>
+          </li>
+        </ul>
+        <ul class="footer-links-menu">
+          <li class="uppercase"><a href="#">events</a></li>
+          <li class="uppercase"><a href="#">about</a></li>
+          <li class="uppercase"><a href="#">advertise</a></li>
+          <li class="uppercase"><a href="#">jobs</a></li>
+          <li class="uppercase"><a href="#">contact</a></li>
+        </ul>
+      </div>
+
+      <div class="footer-copyright">
+        <img
+          width="37"
+          height="37"
+          src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+          alt="TNW footer logo"
+        />
+        <span> &nbsp; © 2006–2020 The Next Web B.V. </span>
+        <span>
+          &nbsp; Made with
+          <img src="assets/heart.svg" width="11" height="11" alt="heart logo" />
+          in Amsterdam. &nbsp;</span
+        >
+        <span>Powered by &nbsp; </span>
+        <img
+          width="65"
+          height="65"
+          src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/stackpath.svg"
+          alt="Stackpath logo"
+        />
+        <p>
+          &nbsp;
+          <a href="http://jigsaw.w3.org/css-validator/check/referer">
+            <img
+              style="border: 0; width: 88px; height: 31px;"
+              src="http://jigsaw.w3.org/css-validator/images/vcss-blue"
```

Comment:
## No inline styling

Using Inline Styles is probably one of the worst CSS practices that many web designers and developers are still doing to this day. It transgresses the main goal of why we use CSS – which is to separate style from the HTML structure. This breaks the specificity of web pages

- [ ] Remove all `style` inline you have in the HTML file

**Note:** Remove all inline style rules in the entire HTML page

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_